sixtyfps_runtime/interpreter/Cargo.toml

```diff
@@ -17,13 +17,34 @@ keywords = ["gui", "toolkit", "graphics", "design", "ui"]
 path = "lib.rs"
 
 [features]
+
+default = ["backend-gl", "x11", "backend-qt"]
+
+## enable the [`print_diagnostics`] function to show diagnostic in the console output
 display-diagnostics = ["sixtyfps-compilerlib/display-diagnostics"]
+
+# (internal) export C++ FFI functions
 ffi = ["spin_on", "sixtyfps-corelib/ffi"]
-x11 = ["sixtyfps-rendering-backend-default/x11"]
-wayland = ["sixtyfps-rendering-backend-default/wayland"]
-backend-gl = ["sixtyfps-rendering-backend-default/sixtyfps-rendering-backend-gl"]
-backend-qt = ["sixtyfps-rendering-backend-default/sixtyfps-rendering-backend-qt"]
-default = ["backend-gl", "x11", "backend-qt"]
+
+## Make use the Rust strandard library.
+std = ["sixtyfps-corelib/std"]
+
+#! ### Backends
+
+## The Qt backend. Uses Qt for the windowing system integration and rendering.
+## This backend is required to use the `native` style.
+## It requires Qt 5.15 or later to be installed. If Qt is not installed, the
+## backend will not be operational
+backend-qt = ["sixtyfps-rendering-backend-default/sixtyfps-rendering-backend-qt", "std"]
+
+## The GL backend uses the `winit` for the windowing system integration,
+## and the `femtovg` crate for the rendering.
+## On unix, at least one of `x11` or `wayland` crate must be enabled.
+backend-gl = ["sixtyfps-rendering-backend-default/sixtyfps-rendering-backend-gl", "std"]
+## Same as the GL backend, with the `x11` feature of `winit` enabled
+x11 = ["sixtyfps-rendering-backend-default/x11", "backend-gl"]
+## Same as the GL backend, with the `wayland` feature of `winit` enabled
+wayland = ["sixtyfps-rendering-backend-default/wayland", "backend-gl"]
```

Comment:
Bummer this can't be shared, but no big deal either. But the above suggestions should probably be applied here to, if applicable :)